src/main/java/entities/Deadline.java

```diff
@@ -0,0 +1,66 @@
+package entities;

```

Comment:
Similarly, could this be placed in a `duke` directory such that the package is named `duke.entities`? Developers might not understand that `entities` package is specifically for the Duke application.